tutorials/6 Analysis/Impedance Analysis.ipynb

```diff
@@ -76,81 +82,93 @@
   },
   {
    "cell_type": "markdown",
+   "id": "lined-azerbaijan",
```

Comment:
Regarding : `"id": "lined-azerbaijan"
`
Please clear cell output, save with python3  name for kernel. 
Same thing throughout the notebook.

Reply:
cells are cleared and saved with python3 already, can you better elaborate the concern?

Reply:
just tried to clear again, no change, it does not get removed. Will keep it.